src/c/urweb.c

```diff
@@ -57,6 +58,50 @@ int uw_really_write(int fd, const void *buf, size_t len) {
   return 0;
 }
 
+int uw_accept(int sockfd, struct sockaddr *addr, socklen_t *addrlen) {
+  static pthread_mutex_t m = PTHREAD_MUTEX_INITIALIZER;
+
+  int ret;
```

Comment:
Please initialize `ret` (I’d say `ret = -1`). The compiler will optimize out the initialization, so there’s no performance impact, and it reduces the risk of undefined behavior.